frontend/src/v5/ui/routes/viewer/ticketFiltersSetter/ticketFiltersSetter.component.tsx

```diff
@@ -15,7 +15,7 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
(originally reported in https://github.com/3drepo/3drepo.io/pull/5492)
Sub pins do not disappeared when go back to tickets list after refresh, using ticket with multiple pins.
![07-refreshSmallPin](https://github.com/user-attachments/assets/b9f89320-aff1-4410-87ac-dbb738d52d55)